lib/solr_ead/component.rb

```diff
@@ -71,7 +71,7 @@ class SolrEad::Component
 
   def to_solr(solr_doc = Hash.new)
     super(solr_doc)
-    Solrizer.insert_field(solr_doc, "ref", self.ref.first.strip, :stored_sortable)
+    Solrizer.insert_field(solr_doc, "ref", self.ref.first.strip, :stored_sortable) if self.ref.respond_to?(:first)
```

Comment:
Is there a case where `self.ref` could return an empty array? (which would cause `strip` to fail on `nil`)